examples/green-house-migration/routes/api/candidates.py

```
@@ -0,0 +1,1388 @@
+        _response = client.get_candidate(candidate_id)
+
+        if "data" not in _response:
+            raise HTTPException(status_code=404, detail="Candidate not found")
+
+        return _normalize_teamtailor_candidate(_response["data"])
+
+    except Exception as e:
+        logger.error("Failed to get candidate %s: %s", candidate_id, e)
+        raise HTTPException(
+            status_code=500, detail=f"Failed to get candidate: {str(e)}"
+        )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)